### DISCUSSION TITLE
✅ Image (and text) swap on hover widget for Elementor pro

### DISCUSSION THREAD

**Discussion by SocialPepper** [elementor/elementor#22012] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The problem is that there is not a simple solution/widget for an image swap on hover in Elementor (pro). What also would be perfect is not only the images can swap on hover, but also text changes on hover.

### Describe the solution you'd like

The solution (by third parties) already exists, like this one: https://www.youtube.com/watch?v=jwkVEsssC4A or this one: https://www.youtube.com/watch?v=3Qd8o0FGx3M

### Describe alternatives you've considered

It can be solved by placing an image widget in a column with a backgroundimage. The image needs to become 100% transparent when hovering over it so that the column background image comes visible. Challenge can be if images are different sizes. An initial used text widget, can nog change/disappear on hover together with an image swap.

### Additional context

On this website, below the large openingphoto you can see the solution: https://beheer-plus.nl/ On that website the hover effect on the 4 images is created with .svg images including text with the procedure described by the "alternatives" above.

**Comment by nicholaszein** [comment]
Hello @SocialPepper!

This is already possible using the Flip Box widget with the **`Fade`** effect!

![image](https://github.com/user-attachments/assets/9dda2692-f963-4277-b1c7-884ddcd51638)

The Flip Box widget was added to Elementor Pro version 1.2.0 in February, 2017.

Kind regards
